### PR TITLE
[TorchFX] Graph tests extended with edge shapes

### PR DESCRIPTION
**Pull request by daniil-lyakhov**
### Changes

Graph tests extended with edges shape

### Reason for changes

To test edge shapes by graph tests

### Related tickets

#2904

### CODE REVIEW

File: tests/torch/data/fx/reference_graphs/extracted_graphs/ConvolutionWithAllConstantInputsModelconv2d_conv2d.dot [hunk 1]
```diff
@@ -0,0 +1,11 @@
+strict digraph  {
+"0 _param_constant0" [id=0, type=get_attr];
+"1 add" [id=1, type=add];
+"2 conv2d_input" [id=2, type=input];
+"3 conv2d" [id=3, type=conv2d];
+"4 output" [id=4, type=output];
+"0 _param_constant0" -> "1 add"  [label="(1, 1, 1, 1)", style=solid];
+"1 add" -> "3 conv2d"  [label="(1, 1, 1, 1)", style=solid];
+"2 conv2d_input" -> "3 conv2d"  [label=None, style=solid];
```
Comment (AlexanderDokuchaev) [inline tests/torch/data/fx/reference_graphs/extracted_graphs/ConvolutionWithAllConstantInputsModelconv2d_conv2d.dot:9]: None for edge from input node is is ok?

File: tests/torch/fx/test_models.py [hunk 2]
```diff
@@ -28,18 +28,28 @@
...
 from nncf.torch.dynamic_graph.patch_pytorch import disable_patching
+from tests.cross_fw.shared.nx_graph import compare_nx_graph_with_reference
 from tests.cross_fw.shared.paths import TEST_ROOT
 from tests.torch import test_models
-from tests.torch.test_compressed_graph import check_graph
 
-FX_DIR_NAME = Path("fx")
-FX_QUANTIZED_DIR_NAME = Path("fx") / "quantized"
+
+def check_fx_graphs(graph: NNCFGraph, path_to_dot: str, graph_dir: str):
```
Comment (AlexanderDokuchaev) [inline tests/torch/fx/test_models.py:43]: Is not fx specific change, please add new arg `extended` to existed function

**Review by AlexanderDokuchaev — CHANGES_REQUESTED** [review]

File: tests/torch/data/fx/reference_graphs/extracted_graphs/ConvolutionWithAllConstantInputsModelconv2d_conv2d.dot (hunk 1 above)
Comment (daniil-lyakhov) [reply tests/torch/data/fx/reference_graphs/extracted_graphs/ConvolutionWithAllConstantInputsModelconv2d_conv2d.dot:9]: Planned in https://github.com/openvinotoolkit/nncf/issues/2934

File: tests/torch/fx/test_models.py (hunk 2 above)
Comment (daniil-lyakhov) [reply tests/torch/fx/test_models.py:43]: All references graphs will be placed back to directory "tests/torch/data/reference_graphs/fx" and other references like metatypes will be placed to "tests/torch/data/fx/reference_metatypes", you ok with that?

Comment (AlexanderDokuchaev) [reply tests/torch/fx/test_models.py:43]: This behavior changes in this PR? Please write about all changes in PR description.

But i suggest firstly discuss about changes in structure of tests on meeting.

Why do you prefer `tests/torch/data/fx` but not `tests/torch/fx/data`?

**Review by AlexanderDokuchaev — CHANGES_REQUESTED** [review]

File: tests/torch/fx/test_models.py (hunk 2 above)
Comment (daniil-lyakhov) [reply tests/torch/fx/test_models.py:43]: I have no preferences, please tell me the right structure and I'll do it

Comment (AlexanderDokuchaev) [reply tests/torch/fx/test_models.py:43]: If you have no preferences about it, lets keep structure as it.
And will discuss about structure on meeting.

**Comment by daniil-lyakhov** [comment]
@AlexanderDokuchaev, I keep the ref structure as is is, please merge if you have no comments

**Review by AlexanderDokuchaev — APPROVED** [review]

**Pull request merged**